meilisearch/client.py

```diff
@@ -205,6 +207,30 @@ def index(self, uid: str) -> Index:
             return Index(self.config, uid=uid)
         raise ValueError("The index UID should not be None")
 
+    def multi_search(self, queries: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Based on your description of how this will work the types should be slightly different here.

```suggestion
    def multi_search(self, queries: List[Dict[str, Any]]) -> Dict[str, List[Dict[str, Any]]:
```

Reply:
Oh, you are right! Thanks